src/formatter/progress_bar_formatter.js

```diff
@@ -38,7 +39,9 @@ export default class ProgressBarFormatter extends Formatter {
   }
 
   logErrorIfNeeded({ sourceLocation, result }) {
-    if (isIssue(result.status)) {
+    const status = result.status
+    const shouldLog = isIssue(status) || status === Status.RETRY
```

Comment:
Can we have isIssue account for the retry status instead of accounting for it here?

Reply:
I didn't want to push the `RETRY` check into`isIssue`, as I guess that may be used to determine whether a test should be treated as a failure and stop execution.
If that's not the case I agree to move the check inside, even though I would still question whether retry at that point is to be considered an issue.